package.json

```diff
@@ -60,13 +60,32 @@
           "light": "./assets/HashiCorp_Icon_Black_RGB.svg",
           "dark": "./assets/HashiCorp_Icon_White_RGB.svg"
         }
+      },
+      {
+        "id": "waypoint_hcl",
+        "aliases": [
+          "HCL"
```

Comment:
```suggestion
          "Waypoint"
```

This is essentially the human readable language identifier in the lower right hand corner of the VS Code editor, so we want this to be as descriptive as possible.

We shouldn't use HCL here because we already have a generic HCL catch all above.

With all the edits, it should look like this:

![image](https://user-images.githubusercontent.com/272569/199793383-64aebea9-1cf7-4280-b004-5fe968e28a61.png)